pkg/sql/pgwire/testdata/pgtest/row_description

```diff
@@ -119,3 +119,21 @@ ReadyForQuery
 {"Type":"DataRow","Values":[{"text":"hello"}]}
 {"Type":"CommandComplete","CommandTag":"SELECT 1"}
 {"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Parse {"Query": "SELECT b FROM tab3"}
+Bind
+DescribePortal
```

Comment:
I'd prefer to keep this as `Describe` and manually specify the object type, ala https://github.com/cockroachdb/cockroach/blob/a0f5c15b3929c224a46e3ca2545171f349c233fa/pkg/sql/pgwire/testdata/pgtest/portals#L37

Reply:
sounds good to me!